extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -87,13 +87,14 @@ export class AnchorAdStrategy {
    * @private
    */
   placeStickyAd_() {
+    const baseAttributes = this.baseAttributes_;
     const viewportWidth = Services.viewportForDoc(this.ampdoc).getWidth();
     const attributes = /** @type {!JsonObject} */ (Object.assign(
       dict(),
-      this.baseAttributes_,
+      baseAttributes,
       dict({
         'width': String(viewportWidth),
-        'height': '100',
+        'height': baseAttributes.height || '100',
```

Comment:
Why is this helpful? I see neither you or other providers are setting the `height` attribute for sticky ads.

Reply:
`baseAttributes.height` allow me to set max-height from `getConfigUrl()` JSON response.